### ISSUE TITLE
Performance issues on Android

### ISSUE THREAD

**Issue by mahmoudatef955** [uxcam/flutter-plugin#26]
When I integrated this SDK into my project I noticed a big effect on app performance on Android devices. We were experiencing janky scrolling an stutter  on Android devices. Everything is perfect when I remove this plugin.

Flutter SDK Version: 2.3.0

Flutter version:
Flutter 3.7.3 • channel stable • https://github.com/flutter/flutter.git
Framework • revision 9944297138 (9 days ago) • 2023-02-08 15:46:04 -0800
Engine • revision 248290d6d5
Tools • Dart 2.19.2 • DevTools 2.20.1

**Comment by ElevateMoney** [comment]
This is also an issue that just appeared recently in [cordova-uxcam](https://github.com/uxcam/cordova-uxcam) plugin we've been using the past year or two. We are on the current version "3.5.2". It also occurs on iOS devices for us. Only solution has been to remove it so far.

**Comment by vjself** [comment]
This is also an issue that just appeared recently in [cordova-uxcam](https://github.com/uxcam/cordova-uxcam) plugin we've been using the past year or two. We are on the current version "3.5.2". It also occurs on iOS devices for us. Only solution has been to remove it so far.

**Comment by mahmoudatef955** [comment]
Yeah seems like removing it is the solution so far

**Comment by tcore** [comment]
+1

**Comment by cepicdalim** [comment]
We have also the same situation, is there any plan to fix it?

**Comment by justincbeck** [comment]
Yes, we are having the same issue. We _really_ want screen recording but this isn't performant enough.

**Comment by gabrieluxcam** [comment]
Hello @mahmoudatef955 Gabriel here from UXCam, thanks for bringing this up. We are aware of the issue, unfortunately it's a complicated problem that requires big changes, and we are working on a long term solution to improve this. 

As for iOS devices @vjself it should be taken care of with the latest cordova 3.6.0, could you give it a try? 

We will keep you updated as soon as possible with any news about progress being made, thanks for your collaboration and patience.

**Comment by mahmoudatef955** [comment]
Hello @gabrieluxcam  thanks for the update.Looking forward for the fix.

**Comment by SamerOrfali22** [comment]
@gabrieluxcam it have been more than one year and still android has performance issues, when this long term solution will be deployed?
